packages/hash/frontend/src/components/layout/PageHeader/HeaderIconButton.tsx

```diff
@@ -1,18 +1,22 @@
 import { styled } from "@mui/material";
 import { IconButton } from "../../IconButton";
 
+const focusBorderOffset = 6;
+const focusBorderWidth = 3;
+
 export const HeaderIconButton = styled(IconButton)<{ open: boolean }>(
-  ({ theme, open }) => ({
-    boxShadow: open
-      ? `0px 0px 0px 2px ${theme.palette.common.white}, 0px 0px 0px 5px ${theme.palette.gray[40]}`
-      : "unset",
+  ({ theme }) => ({
     transition: theme.transitions.create(["box-shadow"]),
     ":hover": {
       boxShadow: `0px 0px 0px 2px ${theme.palette.common.white}, 0px 0px 0px 5px ${theme.palette.gray[40]}`,
     },
-    "&:focus-within": {
-      outline: "none",
-      boxShadow: `0px 0px 0px 2px ${theme.palette.common.white}, 0px 0px 0px 5px ${theme.palette.blue[70]}`,
+    "&:focus-visible:after": {
+      top: -focusBorderOffset,
+      left: -focusBorderOffset,
+      right: -focusBorderOffset,
+      bottom: -focusBorderOffset,
+      borderWidth: focusBorderWidth,
```

Comment:
@benwerner01 this increases the border width and offset for the IconButtons in the `Header`